samples/image-create-and-upload-sample/image_create_and_upload_script.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+source config.sh
+ 
+function standardize_object_name() {
```

Comment:
add a comment with the purpose of this function.

Reply:
Added comment for this method